crates/tx/src/data/mod.rs

```diff
@@ -500,7 +484,7 @@ pub struct VpsResult {
 impl<T: Serialize> fmt::Display for TxResult<T> {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
         if f.alternate() {
-            write!(f, "Transaction is valid. Gas used: {}", self.gas_used)
+            write!(f, "Transaction is valid.")
```

Comment:
My opinion is that keeping this log is important if we are able to still.

Reply:
I've had a chat with @Fraccaman on this. The gas is already emitted as an event, so this field in the `TxResult` was essentially duplicated. I'll try to update the logs in the client to retrieve this piece of information from the events

Reply:
Brought back the logging of the gas used in 209da0c6ad but directly in the SDK when fetching the tx events. It is no longer possible to log it directly in here since the gas is no longer present here